### PR TITLE
Added support for SpamSieve App

### PR DESCRIPTION
**Pull request by michaelgrossklos**
I added a spamsieve.cfg file to enable SpamSieve backup

### CODE REVIEW

File: mackup/applications/spamsieve.cfg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[application]
+name = SpamSieve
+
+[configuration_files]
+Library/Application Support/SpamSieve/
+Library/Preferences/com.c-command.SpamSieve.plist
+
```
Comment (lra) [inline mackup/applications/spamsieve.cfg]: You can remove this empty line

Comment (michaelgrossklos) [reply mackup/applications/spamsieve.cfg]: I did. Thanks for pointing it out.

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,7 +13,8 @@
 - Added support for mitmproxy (via @etiennelb)
 - Added support for Hocus Focus (via @jrolfs)
 - Added support for TripMode (via @ttuygun)
-- Added support for Starship (via @callummr)
+- Added support for Starship (via @callumm)
```
Comment (lra) [inline CHANGELOG.md:16]: Why?

File: README.md [hunk 3]
```diff
@@ -538,6 +538,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
 - [SpaceLauncher](https://spacelauncherapp.com/)
 - [Spacemacs](https://github.com/syl20bnr/spacemacs)
 - [Spark](http://www.shadowlab.org/softwares/spark.php)
+- [SpamSieve](https://www.application-systems.de/spamsieve/)
```
Comment (lra) [inline README.md:541]: Can you keep the alpha order?

File: mackup/applications/spamsieve.cfg [hunk 4]
```diff
@@ -0,0 +1,6 @@
+[application]
+name = SpamSieve
+
+[configuration_files]
+Library/Application Support/SpamSieve/
```
Comment (lra) [inline mackup/applications/spamsieve.cfg:5]: No ending `/` needed.

File: CHANGELOG.md (hunk 2 above)
Comment (michaelgrossklos) [reply CHANGELOG.md:16]: Sorry, just a mistake.

File: README.md (hunk 3 above)
Comment (michaelgrossklos) [reply README.md]: Changed

File: CHANGELOG.md (hunk 2 above)
Comment (lra) [reply CHANGELOG.md:16]: Can you fix it?

Comment (michaelgrossklos) [reply CHANGELOG.md:16]: I did

Comment (lra) [reply CHANGELOG.md:16]: No, the PR is still showing that you've changed `callummr` to `callumm`.

**Pull request closed without merging**